src/helpers.tsx

```diff
@@ -0,0 +1,28 @@
+import { Todo } from './types/Todo';
+
+export const getFilteretTodos = (todos: Todo[],
+  searchQuery: string,
+  status: string) => {
+  let visibleTodos = todos;
+
+  switch (status) {
+    case 'active':
+      visibleTodos = visibleTodos.filter(todo => !todo.completed);
+      break;
+
+    case 'completed':
+      visibleTodos = visibleTodos.filter(todo => todo.completed);
+      break;
+
+    default:
+      break;
+  }
```

Comment:
I think if condition should be here (if (status) {switch...} and case for the status 'all'.

Reply:
If status === 'all' we don`t need to do any filtering, and i think if we add case for `all`, it`ll  be redundant